src/main/java/core/basesyntax/service/FileReaderImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
```

Comment:
Not fixed, Please divide the logic here - reader should read whatever is provided as is, we should separate our concerns. Move parsing data to transactions to a separate service